core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterDMTest.java

```diff
@@ -11590,6 +11590,33 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
     sql(query).withSpark().ok(expected);
   }
 
+  @Test public void testStandardHash() {
+    final RelBuilder builder = relBuilder();
+    final RexNode stdHash = builder.call(SqlLibraryOperators.STANDARD_HASH,
+        builder.scan("EMP").field("ENAME"));
+    final RelNode root = builder
+        .scan("EMP")
+        .project(stdHash)
+        .build();
+    final String expectedBQSql = "SELECT STANDARD_HASH(\"ENAME\")"
```

Comment:
change this to `expectedOracleSql`

Reply:
@nosheenSheikh02 Chaged the name.Please have a look.